README.md

```diff
@@ -12,6 +12,18 @@ This module provides an isomorphic render method for React components in an Expr
 $ npm install --save react-express-middleware react react-dom
 ```
 
+`index.html`
+```html
+<html>
```

Comment:
Not everyone uses ejs.  I would write a paragraph about how to integrate it, saying that "the variable `content` is provided in your view engine, here is an example with ejs:"

And for the example, do you think it is more helpful to have this whole empty html doc? or can we just show `<div id="app"><%- content %></div>`?